bin/fin

```diff
@@ -1991,6 +2006,24 @@ ssh_add () {
 
 #----- Installations and updates -----
 
+# Reports version stats
+stats_ping ()
+{
+	[[ "$DOCKSAL_STATS_OPTOUT" != 0 ]] && return # Don't run if the user opted out
+	[[ "$CI" == "true" ]] && return # Don't run in CI
+
+	# We are passing OS version via a custom User-Agent request header, which GA can parse
+	local user_agent
+	is_linux && user_agent="fin/${FIN_VERSION} (Linux $(lsb_release -si)-$(lsb_release -sr))"
+	is_mac && user_agent="fin/${FIN_VERSION} (Macintosh Intel Mac OS X $(sw_vers -productVersion)))"
+	is_windows && user_agent="fin/${FIN_VERSION} (Windows NT $(uname -s | sed 's/CYGWIN_NT-\(.*\)-WOW/\1/'))"
+
+	curl -kfsL \
```

Comment:
I believe these curl calls should be daemonized to prevent console locking if there's internet slowage or unavailability.
http://stackoverflow.com/a/15944238/1359178